service/src/main/java/org/jboss/sbomer/service/generator/image/rest/v1beta1/SbomsV1Beta1.java

```diff
@@ -97,13 +97,14 @@ public Page<BaseSbomRecord> searchSboms(
             @QueryParam("query") String rsqlQuery,
             @DefaultValue("creationTime=desc=") @QueryParam("sort") String sort) {
 
+        // TODO: Pagination should be done here, not in the service
```

Comment:
wdym?

Reply:
I'll remove this comment.

I couldn't get out of my mind the idea that `Page` is related to the presentation of the list of given resources. What I was thinking is that the service should return entities (or records) which then would be translated into a `Page` object later and not part of the service itself.

Reply:
right, I was thinking more of the query pagination (limiting the query to particular offset and number of results) rather than the presentation, so this is why I was asking how you could do that if not in the service which is responsible of creating the query. since you were thinking about the presentation, ok :-)